### ISSUE TITLE
Crashed on EXPO 48

### ISSUE THREAD

**Issue by rohankm** [bamlab/react-native-flipper-performance-monitor#85]
hi I followed the exact guide for expo projects.. but the app crashed when I start monitoring 

```
Your app just crashed. See the error below.
java.lang.NullPointerException: Attempt to invoke virtual method 'com.facebook.react.bridge.NativeModule com.facebook.react.bridge.ReactContext.getNativeModule(java.lang.Class)' on a null object reference
  com.facebook.react.modules.debug.FpsDebugFrameCallback.<init>(FpsDebugFrameCallback.java:79)
  tech.bam.rnperformance.flipper.FPSMonitor.start(FPSMonitor.java:23)
  tech.bam.rnperformance.flipper.RNPerfMonitorPlugin$3.run(RNPerfMonitorPlugin.java:86)
  android.os.Handler.handleCallback(Handler.java:938)
  android.os.Handler.dispatchMessage(Handler.java:99)
  android.os.Looper.loopOnce(Looper.java:210)
  android.os.Looper.loop(Looper.java:299)
  android.app.ActivityThread.main(ActivityThread.java:8213)
  java.lang.reflect.Method.invoke(Native Method)
  com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:556)
  com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1045)
```

**Comment by Almouro** [comment]
Hi @rohankm, 
I've just published v0.4.0 of the react native plugin, could you retry with that version? 🙏

**Comment by roo12312** [comment]
> Hi @rohankm, I've just published v0.4.0 of the react native plugin, could you retry with that version? 🙏

hi its working fine now :)

**Comment by danieldanielecki** [comment]
Please close the issue then.
